File: backend/app/domain/model/fixed_effects_result.py
from dataclasses import dataclass

@dataclass(frozen=True)
class FixedEffectsResult:
    nobs: int
    params: dict[str, float]
    std_errors: dict[str, float]
    tstats: dict[str, float]
    pvalues: dict[str, float]
    rsquared_within: float
    rsquared_between: float
    rsquared_overall: float
    dropped_vars: list[str]

File: frontend/src/types/fixedEffectsResult.ts
export type FixedEffectsResult = {
  nobs: number
  params: Record<string, number>
  std_errors: Record<string, number>
  tstats: Record<string, number>
  pvalues: Record<string, number>
  rsquared_within: number
  rsquared_between: number
  rsquared_overall: number
}
